nextflow/integration.config

```diff
@@ -0,0 +1,207 @@
+// nextflow.config — Integration
+process.executor = 'local'
+
+params {
+
+  // Base params
+  preprocessed_obj = '/teaseq_unfilt.h5mu'
+  raw_obj = "/teaseq.h5mu"
+  outdir = './results'
+  mode   = 'integration'
+  sample_prefix = 'teaseq'
+  lisi_run = true
+
+  // Trying to implement resources here
+  resources = [
+    threads_high : 1,
+    threads_medium: 1,
+    threads_low  : 1,
+    threads_gpu  : 2
+  ]
+
+  // RNA modality
+  rna = [
+    run    : true,
+    tools  : 'bbknn,scvi,scanorama',
+    column : 'dataset',
+    harmony: [ sigma: 0.1, theta: 1.0, npcs: 30 ],
+    bbknn  : [ neighbors_within_batch: 20 ],
+    scvi   : [
+      seed: 1492,
+      exclude_mt_genes: true,
+      mt_column: 'mt',
+      model_args   : [ n_layers: null, n_latent: null, gene_likelihood: 'zinb' ],
+      training_args: [ max_epochs: 400, train_size: 0.9, early_stopping: true ],
+      training_plan: [ lr: 0.001, n_epochs_kl_warmup: 400, reduce_lr_on_plateau: true,
+                       lr_scheduler_metric: null, lr_patience: 8, lr_factor: 0.1 ]
+    ],
+    neighbors: [ npcs: 30, k: 30, metric: 'euclidean', method: 'scanpy' ]
+  ]
+
+  // Protein modality
+  prot = [
+    run    : true,
+    tools  : 'harmony',
+    column : 'orig.ident',
+    harmony: [ sigma: 0.1, theta: 1.0, npcs: 30 ],
+    bbknn  : [ neighbors_within_batch: 20 ],
+    neighbors: [ npcs: 30, k: 30, metric: 'euclidean', method: 'scanpy' ]
+  ]
+
+  // ATAC modality
+  atac = [
+    run   : true,
+    dimred: 'LSI',
+    tools : 'harmony',
```

Comment:
add the list of available tools for each modality (as commment) and check that they can be read when specified in the "tools" parameter